1907-05-07-p1.xml

```diff
@@ -0,0 +1,1140 @@
+                                >May 20</date></p>
+                        <table>
+                            <row>
+                                <cell rows="4">Passage</cell>
+                                <cell>Naples</cell>
+                                <cell>1st Class</cell>
+                                <cell>£ 9.00.</cell>
+                                <cell>2nd Class</cell>
+                                <cell>£ 7.00.</cell>
+                                <cell>3rd Class</cell>
+                                <cell>£ 4.00.</cell>
+                            </row>
+                            <row>
+                                <cell>Marseilles</cell>
+                                <cell>1st Class</cell>
+                                <cell>£ 13.00.</cell>
+                                <cell>2nd Class</cell>
+                                <cell>£ 9.00.</cell>
+                                <cell>3rd Class</cell>
+                                <cell>£ 5.00</cell>
+                            </row>
+                            <row>
+                                <cell>Gibraltar</cell>
+                                <cell>1st Class</cell>
+                                <cell>£ 15.00.</cell>
+                                <cell>2nd Class</cell>
+                                <cell>£ 9.00.</cell>
+                                <cell>3rd Class</cell>
+                                <cell>£ 5.00.</cell>
+                            </row>
+                            <row>
+                                <cell>Plymouth or Tilbury</cell>
+                                <cell>1st class</cell>
+                                <cell>£ 19.00.</cell>
+                                <cell>2nd class</cell>
+                                <cell>£ 12.00.</cell>
+                                <cell>3rd class</cell>
+                                <cell>£ 9.00.</cell>
+                            </row>
+                        </table>
+                        <p>The issue of return tickets to and from Egypt has been discontinued.
+                            Passengers paying full fare one direction will, however, be allowed
+                            abatement of one-third off fare back if return voyage be made within
+                            four months of arrival, or abatement of 20 o/o if return voyage be made
+                            within six months of arrival.</p>
+                        <p>Passengers returning by the Line obtain one-third rebate off the above
+                            fares if leaving England before the end of October.</p>
+                        <p>Agents. Cairo:—Thos. Cook &amp; Son. Alexandria : —R. J. Moss &amp;
+                            Co.—For all particulars apply</p>
+                        <p>Wm. STAPLEDON &amp; Sons, PORT SAID and PORT-TEWFIK (Suez). 31-12-906</p>
+                    </div>
+                </div>
```

Comment:
This `</div>` tag, which closes the advertisement section, should move just before the `</div>` that closes the page.